minio/datatypes.py

```diff
@@ -815,6 +815,9 @@ def __init__(self, func):
 
     def _close_response(self):
         """Close response."""
+        if self._stream:
+            self._stream.close()
+            self._stream = None
         if self._response:
```

Comment:
This closes underneath stream as well. I would need to check your `urllib3`

Reply:
Thank you for the pointer.